src/main/java/uk/gov/hmcts/reform/ccd/documentam/controller/advice/CaseDocumentControllerAdvice.java

```diff
@@ -71,17 +71,22 @@
 
     @ExceptionHandler(MethodArgumentNotValidException.class)
```

Comment:
Same behaviour in 2 different methods, can we not have 1 method and multiple exception handlers?

`@ExceptionHandler({MethodArgumentNotValidException.class, MissingServletRequestParameterException.class})`